sources/packages/backend/libs/utilities/src/integration-utils.ts

```diff
@@ -8,7 +8,9 @@ export function getTotalYearsOfStudy(completionYears: string): number {
   switch (completionYears) {
     case "12WeeksToLessThan1Year":
       return 1;
-    case "1YearToLessThan2Years":
+    case "52WeeksTo59Weeks":
```

Comment:
The only effect of this change, will be in the generation of ecert and the IER12 files. Did we take this into account in the AC ? that the change will have different numbers generated in the file generation?

Reply:
@guru-aot Below is the AC from the business confirming that.

<img width="541" alt="image" src="https://github.com/bcgov/SIMS/assets/7859295/e4e223fe-19fc-46e0-83f6-bcdcf2b3ed0e">